src/main/java/studio/blockops/vyom/core/utils/RLP.java

```diff
@@ -0,0 +1,272 @@
+package studio.blockops.vyom.core.utils;
```

Comment:
Try to look at **org.nem.core.serialization** package and see how they have implemented their serialization.
They have better implementation of encoding/decoding. Can use their pattern and ethereum RLP encoding technique.